components/AnimatedView/types.ts

```diff
@@ -0,0 +1,20 @@
+export type AnimatedViewProps = {
+  animatedChildHideStyle?: [] | {};
+  animatedChildShowStyle?: [] | {};
+  children?: JSX.Element;
+  customSpinnerComponent?: any;
+  height: number;
+  size: number;
+  spinnerColor: string;
+  spinnerType:
+    | "barindicator"
+    | "dotindicator"
+    | "materialindicator"
+    | "pacmanindicator"
+    | "pulseindicator"
+    | "skypeindicator"
+    | "uiactivityindicator"
+    | "waveindicator";
+  indicatorCount: number;
+  spinnerOptions: {};
+};
```

Comment:
Create the common type of Animatable & Animated types and the intersection of them